text/0000-rfc-process-update.md

```diff
@@ -0,0 +1,166 @@
+---
+tags: ember, ember-cli
+---
+
+* Start Date: 2018-02-04
+* RFC PR: https://github.com/emberjs/rfcs/pull/300
+* Ember Issue: (leave this empty)
+
+# RFC Process Update
+
+## Summary
+
+Introduce the idea of Core Champion to the Ember.js RFC process, merge Ember.js and Ember CLI RFCs repositories,
+and add metadata to make cross-project RFCs easier.
+
+## Motivation
+
+Inspired by Rust, Ember implemented a Request For Commits (RFC) process with the intent to involve the community in discussing the future of the framework.
+The RFC process has proved invaluable.
+Proposals by both Core and community members are discussed and refined,
+with the result coming out much stronger.
+Other projects like React also adopted an RFC process, lending external validation to this idea.
+
+After having been involved with the RFC process for some time, and from feedback from the community, some issues with the current implementation of the process have been noted.
```

Comment:
Following the success of its initial implementation, the RFC process can be further refined to address some issues identified by the community and core team members.